admin/updates/20190508-submission-offsets-1.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+
+ALTER TABLE lowlevel ADD COLUMN submission_offset INTEGER;
+
+-- Updating the submission offset column based on determination of offset
```

Comment:
Doing this in a single query is a good idea, but this will block the entire table and stop other insert actions from taking place. Unfortunately we're going to have to do it in a script which is a bit more complex and will take longer to run, but will leave the site up while it runs.

Take a look at making a script like manage.py that uses a click `cli` in order to start up the application and connect to the database:
https://github.com/metabrainz/acousticbrainz-server/blob/f9afa4017c49b7d412b6557b68744d8fbbb2da07/manage.py#L24-L29

Do a query like:
`select id, gid from lowlevel where submission_offset is null limit 10000`

the limit can be a parameter in the script that we can change

Use something like our offset counting code in the dumps to find the current max offset for each mbid:
https://github.com/metabrainz/acousticbrainz-server/blob/e8fceb76b434bb97aabf1b006e087097b2f66089/db/dump.py#L546-L555
this query might be good:
`select gid, max(submission_offset) from lowlevel where submission_offset is not null group by gid`

then you can loop through the set of 10000 items, and insert the offset counts. Do this loop in a transaction.

You don't need to write the sql in the db module, just write it inline in this script.

Make sure that you print some debugging information - each time it loops around, each time it starts and finishes doing the inserts, how many items have been done and how many remain.